android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -44,14 +45,17 @@ internal const val CACHE_TOP_SITES_KEY = "tiles"
  */
 class ContileTopSitesProvider(
     context: Context,
+    val isMozillaOnline: Boolean = false,
```

Comment:
Could we also reorder the list of parameter here by moving this new parameter to the bottom of the list? We should order non-optional parameters before optional parameters.